### PR TITLE
Instructions review

### PR DESCRIPTION
**Pull request by lucie-docs**
Here is my review of the instructions 🙂

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,15 +17,15 @@ This is an exercise to check your knowledge on using secrets in GitHub Actions w
 Please complete the instructions below:
 
 1. Create your own copy of this repository by using the [Use this template](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-from-a-template#creating-a-repository-from-a-template) button.
-2. Create a new, or use an existing Personal Access Token that has the `repo` scope enabled.
+2. If you don't have any [Personal Access Token](https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/creating-a-personal-access-token), create one with the `repo` scope selected. If you already have one or multiple Personal Access Tokens, skip to step 3.
```
Comment (hectorsector) [inline README.md]:
```suggestion
2. If you don't have any [Personal Access Tokens](https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/creating-a-personal-access-token), create one with the `repo` scope selected. If you already have one or multiple Personal Access Tokens, skip to step 3.
```

Comment (mattdavis0351) [inline README.md]:
```suggestion
2. If you don't have any [Personal Access Tokens](https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/creating-a-personal-access-token), create one with the `repo` scope selected. If you already have one or multiple Personal Access Tokens that are properly scoped, skip to step 3.
```

**Review by hectorsector — APPROVED** [review]
✨ thanks @lucie-docs

**Comment by mattdavis0351** [comment]
@lucie-docs looks good to me

**Pull request merged**